finish/src/main/java/io/openliberty/guides/rest/PropertiesResource.java

```diff
@@ -13,28 +13,21 @@
 package io.openliberty.guides.rest;
 
 import java.util.Properties;
```

Comment:
I get an IDE warning on this line because the `import java.util.Properties;` is not used.  Same for the one in the `start` folder.